### ISSUE TITLE
Typo in the romanian CurrencyTransformer

### ISSUE THREAD

**Issue by ethmelly** [kwn/number-to-words#6]
\Kwn\NumberToWords\Language\Romanian\Transformer\CurrencyTransformer.php at line 53 
`$ret .= ' ' . $this->toWords($fraction, $currNouns[1]);` should actually be 
`$ret .= ' ' . $this->transformer->toWords($fraction, $currNouns[1]);`

**Comment by kwn** [comment]
Hi @ethmelly 

Once I update that with your suggestion I get some errors during tests:

```
1) Kwn\NumberToWords\Language\Romanian\Transformer\CurrencyTransformerTest::testToWords with data set #4 (Kwn\NumberToWords\Language\Romanian\Transformer\CurrencyTransformer Object (...), 'un leu și patruzeci și cinci de lei', 1.45)
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'un leu și patruzeci și cinci de lei'
+'un leu și patruzeci și cinci de bani'
```

I don't really know if it should be "de lei" or "de bani". My colleague is Romanian, so I'll ask her tomorrow and I'll be able to confirm that.

Or if you want, you can create a pull request to fix that. I'm happy to review and merge it.

**Comment by ethmelly** [comment]
hi @kwn 
"de lei" => is like "dollars"
"de bani" => is like "cents"

This is why i suggested the typo , if u further inspect the original code in any php IDE you will see that you are calling the toWords() method with two parameters. I didn't use the testToWords but most likely is failling for same reason ( expected output having same typo in syntax )

**Comment by kwn** [comment]
Hello @ethmelly 

Yes, I totally agree. I had a chat with my colleague and she confirmed the tests were incorrect. 

To be honest I've copypasted Romanian language from another library, so that's why code looks quite ugly. It may also have some bugs because there were just a few test cases.

I've merged your PR. Thank you for your contribution :) :+1:

**Comment by kwn** [comment]
Fixed in https://github.com/kwn/number-to-words/pull/7

Released in 0.0.3
